runtime/jcl/common/system.c

```diff
@@ -462,6 +462,15 @@ jobject getPropertyList(JNIEnv *env)
 			}
 		}
 	}
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	else {
+		/* If a checkpoint can be taken we want to fall back
+		 * to getting the username from the USER env var instead.
+		 */
+		result = 1;
```

Comment:
```suggestion
		result = -1;
```
`j9sysinfo_get_username` returns `0` on success, the number of bytes required to hold the information if the output buffer was too small, `-1` on failure.
I also think this can be moved before https://github.com/eclipse-openj9/openj9/blob/d0a6950ce2d69d89f321c38a69fbf51fe8c3306b/runtime/jcl/common/system.c#L442